recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
> No @jakirkham.

*No* you won't try or *No* that tarball does not work for you? If the former I agree with @jakirkham it is a hard merge... If the latter let us know why. I am OK merging development versions with the right justification.